app-games/src/main/java/com/aptoide/android/aptoidegames/installer/presentation/ProgressText.kt

```diff
@@ -77,6 +78,8 @@ private fun ProgressTextContent(
         overflow = TextOverflow.Ellipsis,
         maxLines = 1
       )
+    }else{
```

Comment:
In AppRows with app titles with two lines, the rating won't appear, as it doesn't have the modifier (With it, it isn't clean as well)